src/components/talk/index.js

```diff
@@ -50,6 +50,22 @@ class LightningTalksPage extends Component {
         }
     }
 
+    hasDuplicates = (talkSubset = [], talkSuperset = []) => {
+        if (talkSubset.length < 1 || talkSuperset.length < 1) return false;
+
+        let hasDuplicate = false;
+        talkSubset = talkSubset.map(talk => talk.id);
```

Comment:
you can make a Set out of talkSubset, e.g. `talkSubset = new Set(talkSubset.map(talk => talk.id)` to make the check for duplication a constant-time op. the method for checking if something exists in a set is `has()`

Reply:
good looks